src/views/product-downloads-view/index.tsx

```diff
@@ -89,7 +89,7 @@ const ProductDownloadsViewContent = ({
 				versionSwitcherOptions={versionSwitcherOptions}
 			/>
 			<OfficialReleasesSection />
-			{featuredLearnCards ? (
+			{featuredLearnCards && featuredLearnCards.length > 0 ? (
```

Comment:
Empty array is truthy, so added this check to prevent rendering a `Feature Tutorials` heading with no items below it.